docs/ref/models/fields.txt

```diff
@@ -2289,7 +2289,8 @@ concrete model and are not relative to the abstract model's ``app_label``:
     :caption: ``production/models.py``
 
     from django.db import models
-    from products.models import AbstractCar
```

Comment:
Leave alone. `AbtractCar` is captioned as `products/models.py` whereas this file is `production/models.py`.

Reply:
ok